src/main/resources/template-ls.json

```diff
@@ -0,0 +1,83 @@
+{
+  "languageServers": [
+    {
+      "name": "CSS Language Server",
+      "instructions": [
+        "To consume CSS language server you need to:",

```

Comment:
You can use Visual Studio Code's CSS language server by following these instructions: